jobs/config.json

```diff
@@ -12611,6 +12621,7 @@
     "args": [
       "--build",
       "--cluster=",
+      "--env=TEST_CLUSTER_LOG_LEVEL=--v=2",
```

Comment:
I think there's also a 500-node gce presubmit job. Can you also set this there?
In general, can you check if there are some other jobs missing this change?

Reply:
There's also 2k-node presubmit job.

I'm curious what's the logic you're using when moving those flags? I would expect sth like find all jobs which are using a given env file and set it for all those.

Reply:
It wasn't set for that then. But sure setting.

Reply:
> I'm curious what's the logic you're using when moving those flags? I would expect sth like find all jobs which are using a given env file and set it for all those.

I'm looking into config files and moving from those to appropriate jobs.

This is exactly the what I said to you offline, that these configs are completely different. There wasn't verbosity flag in 2000-node jobs.